zanata-model/src/main/java/org/zanata/model/HAccount.java

```diff
@@ -58,7 +58,7 @@
  *
  */
 @Entity
-@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+@Cache(usage = CacheConcurrencyStrategy.TRANSACTIONAL)
```

Comment:
We do. Last time I tried to run it, it threw an error and said something
along the lines of "Infinispan only supports Transactional cache".

Carlos A. Munoz
Associate Manager, Software Engineering
Globalization
Red Hat

On Fri, Mar 11, 2016 at 3:58 PM, Sean Flanigan <notifications@github.com>
wrote:

> In zanata-model/src/main/java/org/zanata/model/HAccount.java
> <https://github.com/zanata/zanata-server/pull/1111#discussion_r55792404>:
>
> > @@ -58,7 +58,7 @@
> >   *
> >   */
> >  @Entity
> > -@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
> > +@Cache(usage = CacheConcurrencyStrategy.TRANSACTIONAL)
>
> I'm not sure if we really need to be specifying @Cache(usage =
> CacheConcurrencyStrategy.TRANSACTIONAL) everywhere, but if so we've
> probably missed a few because this code was sidelined for a year.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1111/files#r55792404>.
>